lib/ex_docker_build/docker_build.ex

```diff
@@ -187,4 +196,46 @@ defmodule ExDockerBuild.DockerBuild do
       {:ok, value} -> {:exec_form, value}
     end
   end
+
+  # TODO:
+  defp copy_from_other_container(name, origin, dest, context) do
+    # with true do
```

Comment:
maybe remove function

Reply:
this is a work in progress, i'm going to work on it today